x/fundraising/keeper/auction.go

```diff
@@ -259,6 +256,8 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
 			sdk.NewAttribute(types.AttributeKeyStartTime, auction.GetStartTime().String()),
 			sdk.NewAttribute(types.AttributeKeyEndTime, msg.EndTime.String()),
 			sdk.NewAttribute(types.AttributeKeyAuctionStatus, auction.GetStatus().String()),
+			sdk.NewAttribute(types.AttributeKeyMinBidPrice, msg.MinBidPrice.String()),
```

Comment:
Please let me know the necessity of this change. I just added in order to fill this with additional variables.

Reply:
Yes, I think it is necessary to emit `MinBidPrice` event.